plugins/dev-tools/README.md

```diff
@@ -47,5 +47,24 @@ addGUIControls((options) => {
 }, defaultOptions);
 ```
 
+### Populate Random
+
+The `populateRandom` function adds random blocks to a workspace. Blocks are selected from the full set of defined blocks. Pass in a worskpace and how many blocks should be created.
```

Comment:
Similar to other comment, we would want this to be a default button in the playground, but I'm not opposed to exporting it anyway. (as it could be used in a different setting)

Reply:
The idea is to export it for use outside of our playgrounds. However, I don't want to add the button function and then remove it immediately, so I'm just going to take it out of this PR. It already served its purpose, which was in testing.

Reply:
Makes sense.